backend/src/routes.js

```diff
@@ -1,12 +1,32 @@
-const express = require('express')
+const express = require('express');
+
```

Comment:
I think this is also missing the import for the Users entity that you can get after merging #16. ;)